katz/src/main/kotlin/katz/data/Option.kt

```diff
@@ -31,6 +31,7 @@ sealed class Option<out A> : OptionKind<A> {
     companion object : OptionMonad, GlobalInstance<Monad<Option.F>>() {
         inline fun <reified A : Any> fromNullable(a: A?): Option<A> = if (a != null) Option.Some(a) else Option.None
         operator fun <A> invoke(a: A): Option<A> = Option.Some(a)
+        inline fun <reified T: Any> empty() = Option.fromNullable<T>(null)
```

Comment:
I wouldn't say so. Option.None is now being created as an Option<Nothing> which is not the same that Option<A> with a null value. I was using this method for `Foldable#find`, but seems like I finally desisted and it's not being used anymore. So removing it for now.